src/View/pagination.html.twig

```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+  <title>Message Board</title>
+</head>
+<body>
```

Comment:
it might also be worth considering moving this file into a directory specifically for includes so that it doesn't get mixed up with the page templates. Something like src/View/includes/pagination.html.twig Then you include it in twig like this:
```
{{ include('includes/pagination.html.twig',{
   ...
}) }}
```